nominees/nomie.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Nomie",
+  "description": "An open source life-tracker app",
+  "website": "https://nomie.app/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/open-nomie/nomie5/blob/v5-develop/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "This app has a great privacy policy. It doesn't collect, use, sell or track personal data. Users can store their data locally or encrypted in Blockstack app",
```

Comment:
@Rolikasi I'm still not convinced how this `evidenceText` supports `SDG 3`: "Good Health and Well-being". Kindly provide stronger relevance or look for another `SDG` this project aligns with. Thanks.

Reply:
Thanks for the review @nathanbaleeta. I updated evidenceText, take a look at it. I'm sure that this project is about good health and well-being)